src/view/com/util/List.tsx

```diff
@@ -49,11 +64,35 @@ function ListImpl<ItemT>(
     },
   })
 
+  let refreshControl
+  if (refreshing !== undefined || onRefresh !== undefined) {
+    refreshControl = (
+      <RefreshControl
+        refreshing={refreshing ?? false}
+        onRefresh={onRefresh}
+        tintColor={pal.colors.text}
+        titleColor={pal.colors.text}
+        progressViewOffset={headerOffset}
+      />
+    )
+  }
+
+  let contentOffset
+  if (headerOffset != null) {
+    style = addStyle(style, {
+      paddingTop: headerOffset,
```

Comment:
Note: this overrides it. This is a bit blunt but maybe also more obvious when it's wrong than if I tried to sum them up or something. I decided throwing on it might be a bit much. Maybe could console warn.